.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
I believe this may conflict with Yarn's default package manager, which is `https://registry.yarnpkg.com`. So we may end up with mixed URLs in `yarn.lock`.

FWIW I use [npmrc](https://www.npmjs.com/package/npmrc) to switch between global npm registries.

Reply:
Hmm, interestingly I don't see any mixed URLs in `yarn.lock`, so maybe I'm wrong about this. It's using `https://registry.yarnpkg.com` everywhere.

Reply:
Yeah weird, even deleting the `yarn.lock` and re-running `yarn` still uses `registry.yarnpkg.com`. Looks like it's a yarn v1 thing: https://github.com/yarnpkg/yarn/issues/3612

Reply:
yeah, I tested it even after `yarn cache clean`, `yarn install` the yarn.lock only has `https://registry.yarnpkg.com`. For safety, the npmrc setting has been update to `https://registry.yarnpkg.com`